libraries/SD/src/sd_diskio.cpp

```diff
@@ -671,8 +671,10 @@ uint8_t sdcard_uninit(uint8_t pdrv) {
   if (pdrv >= FF_VOLUMES || card == NULL) {
     return 1;
   }
-  AcquireSPI lock(card);
-  sdTransaction(pdrv, GO_IDLE_STATE, 0, NULL);
+  {
+    AcquireSPI lock(card);
+    sdTransaction(pdrv, GO_IDLE_STATE, 0, NULL);
+  }
```

Comment:
Can you please add a comment, so it's more clear why is there the nested block? Thanks
```suggestion
  {
    AcquireSPI lock(card);
    sdTransaction(pdrv, GO_IDLE_STATE, 0, NULL);
  } // lock is destructed here
```